CHANGELOG.md

```diff
@@ -1,5 +1,14 @@
 # bedrock-vc-delivery ChangeLog
 
+## 3.3.0 - 2023-08-TBD
+
+### Added
+- Add `vc-api delivery` and ` VC-API delivery + DID authn` tests to issue
+  using `generic` credential template.
+
+### Changed
+- Make `expectedCredentialRequests` optional in `openIdExchangeOptions` schema.
```

Comment:
This shouldn't be optional -- can we make it required again?

Reply:
Reverted back the changes [1c88e8b](https://github.com/digitalbazaar/bedrock-vc-delivery/pull/30/commits/1c88e8b745010368495b1378ab8848b9f9313ed3)